core/proguard-consumer-rules.pro

```diff
@@ -57,4 +52,11 @@
 
 # Required for DBEncryption feature using SQLCipher
 -keep class net.sqlcipher.** { *; }
--keep class net.sqlcipher.database.* { *; }
\ No newline at end of file
+-keep class net.sqlcipher.database.* { *; }
```

Comment:
We have migrated to newer sql cipher, which means, the package has changed to
net.zetetic.database.sqlcipher
Also, if this is kept, can you check if the following rules are needed?

Reply:
yes. since we are not using this.